build.cake

```diff
@@ -144,13 +154,13 @@ Task("Copy-Files")
     .IsDependentOn("Run-Unit-Tests")
     .Does(() =>
 {
-    // .NET 4.5
+    // .NET 4.6.2
     DotNetCorePublish("./src/Cake", new DotNetCorePublishSettings
     {
-        Framework = "net45",
+        Framework = "net462",
         VersionSuffix = parameters.Version.DotNetAsterix,
         Configuration = parameters.Configuration,
-        OutputDirectory = parameters.Paths.Directories.ArtifactsBinNet45,
+        OutputDirectory = parameters.Paths.Directories.ArtifactsBinFullFx,
         Verbosity = DotNetCoreVerbosity.Minimal
```

Comment:
@patriksvensson Currently getting an error with copy files on Linux
```
/usr/share/dotnet/sdk/1.0.4/Microsoft.Common.CurrentVersion.targets(1111,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.6.2" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [/cake/src/Cake/Cake.csproj]
An error occurred when executing task 'Copy-Files'.
```

Adjusting copy files task like below seems to sort it
```diff
    // .NET 4.6.2
    DotNetCorePublish("./src/Cake", new DotNetCorePublishSettings
    {
        Framework = "net462",
        VersionSuffix = parameters.Version.DotNetAsterix,
        Configuration = parameters.Configuration,
        OutputDirectory = parameters.Paths.Directories.ArtifactsBinFullFx,
-        Verbosity = DotNetCoreVerbosity.Minimal,
+       MSBuildSettings = msBuildSettings
    });

    // .NET Core
    DotNetCorePublish("./src/Cake", new DotNetCorePublishSettings
    {
        Framework = "netcoreapp1.0",
        Configuration = parameters.Configuration,
        OutputDirectory = parameters.Paths.Directories.ArtifactsBinNetCore,
-        Verbosity = DotNetCoreVerbosity.Minimal,
+       MSBuildSettings = msBuildSettings
    });
```